avm/res/databricks/workspace/main.bicep

```diff
@@ -288,8 +288,8 @@ resource workspace_roleAssignments 'Microsoft.Authorization/roleAssignments@2022
   scope: workspace
 }]
 
-module workspace_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.3.1' = [for (privateEndpoint, index) in (privateEndpoints ?? []): {
-  name: '${uniqueString(deployment().name, location)}-KeyVault-PrivateEndpoint-${index}'
+module workspace_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.3.3' = [for (privateEndpoint, index) in (privateEndpoints ?? []): {
```

Comment:
@clintgrove  I bumped the PE module version to the latest version at the time I updated the module, which was `0.3.3`. I did not update the version.json because I thought a patch number increase is sufficient for this PR. Do you agree?

Reply:
@tyconsulting @clintgrove - Yep this should be fine as the change is fully backward compatible